pkg/engine/template_generator.go

```diff
@@ -141,6 +141,9 @@ func (t *TemplateGenerator) getTemplateFuncMap(vm *api.APIModel) template.FuncMa
       }`
 			return fmt.Sprintf(sshTempl, strings.Join(keyData, ",\n"))
 		},
+		"GetDefaultVMSize":func() string {
+			return api.GetDefaultVMSize(vm.VMConfigurator)
+		},
 		"GetAllowedVMSizes": func() string {
```

Comment:
We don't need two methods here. I suggest changing "GetAllowedVMSizes" to "GetVMSizes" and return both default and allowed values (see my previous comments)